package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "classnames": "^2.3.1",
     "gray-matter": "^4.0.3",
+    "luxon": "^2.0.2",
```

Comment:
not 2.0.3 as @types/luxon????

Reply:
nope, 2.0.2 is the newest version: https://www.npmjs.com/package/luxon